packages/manager/modules/sign-up/src/form/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,642 @@
+  "sign_up_form_enum_IT_area_OT": "Olbia-Tempio (Sardegna)",
+  "sign_up_form_enum_IT_area_OR": "Oristano (Sardegna)",
+  "sign_up_form_enum_IT_area_PD": "Padova (Veneto)",
+  "sign_up_form_enum_IT_area_PA": "Palermo (Sicilia)",
+  "sign_up_form_enum_IT_area_PR": "Parma (Emilia-Romagna)",
+  "sign_up_form_enum_IT_area_PV": "Pavia (Lombardia)",
+  "sign_up_form_enum_IT_area_PG": "Perugia (Umbria)",
+  "sign_up_form_enum_IT_area_PU": "Pesaro e Urbino (Marche)",
+  "sign_up_form_enum_IT_area_PE": "Pescara (Abruzzo)",
+  "sign_up_form_enum_IT_area_PC": "Piacenza (Emilia-Romagna)",
+  "sign_up_form_enum_IT_area_PI": "Pisa (Toscana)",
+  "sign_up_form_enum_IT_area_PT": "Pistoia (Toscana)",
+  "sign_up_form_enum_IT_area_PN": "Pordenone (Friuli-Venezia Giulia)",
+  "sign_up_form_enum_IT_area_PZ": "Potenza (Basilicata)",
+  "sign_up_form_enum_IT_area_PO": "Prato (Toscana)",
+  "sign_up_form_enum_IT_area_RG": "Ragusa (Sicilia)",
+  "sign_up_form_enum_IT_area_RA": "Ravenna (Emilia-Romagna)",
+  "sign_up_form_enum_IT_area_RC": "Reggio Calabria (Calabria)",
+  "sign_up_form_enum_IT_area_RE": "Reggio Emilia (Emilia-Romagna)",
+  "sign_up_form_enum_IT_area_RI": "Rieti (Lazio)",
+  "sign_up_form_enum_IT_area_RN": "Rimini (Emilia-Romagna)",
+  "sign_up_form_enum_IT_area_RM": "Roma (Lazio)",
+  "sign_up_form_enum_IT_area_RO": "Rovigo (Veneto)",
+  "sign_up_form_enum_IT_area_SA": "Salerno (Campania)",
+  "sign_up_form_enum_IT_area_VS": "Medio Campidano (Sardegna)",
+  "sign_up_form_enum_IT_area_SS": "Sassari (Sardegna)",
+  "sign_up_form_enum_IT_area_SV": "Savona (Liguria)",
+  "sign_up_form_enum_IT_area_SI": "Siena (Toscana)",
+  "sign_up_form_enum_IT_area_SR": "Siracusa (Sicilia)",
+  "sign_up_form_enum_IT_area_SO": "Sondrio (Lombardia)",
+  "sign_up_form_enum_IT_area_TA": "Taranto (Puglia)",
+  "sign_up_form_enum_IT_area_TE": "Teramo (Abruzzo)",
+  "sign_up_form_enum_IT_area_TR": "Terni (Umbria)",
+  "sign_up_form_enum_IT_area_TO": "Torino (Piemonte)",
+  "sign_up_form_enum_IT_area_OG": "Ogliastra (Sardegna)",
+  "sign_up_form_enum_IT_area_TP": "Trapani (Sicilia)",
+  "sign_up_form_enum_IT_area_TN": "Trento (Trentino-Alto Adige)",
+  "sign_up_form_enum_IT_area_TV": "Treviso (Veneto)",
+  "sign_up_form_enum_IT_area_TS": "Trieste (Friuli-Venezia Giulia)",
+  "sign_up_form_enum_IT_area_UD": "Udine (Friuli-Venezia Giulia)",
+  "sign_up_form_enum_IT_area_VA": "Varese (Lombardia)",
+  "sign_up_form_enum_IT_area_VE": "Venezia (Veneto)",
+  "sign_up_form_enum_IT_area_VB": "Verbano-Cusio-Ossola (Piemonte)",
+  "sign_up_form_enum_IT_area_VC": "Vercelli (Piemonte)",
+  "sign_up_form_enum_IT_area_VR": "Verona (Veneto)",
+  "sign_up_form_enum_IT_area_VV": "Vibo Valentia (Calabria)",
+  "sign_up_form_enum_IT_area_VI": "Vicenza (Veneto)",
+  "sign_up_form_enum_IT_area_VT": "Viterbo (Lazio)",
+  "sign_up_form_enum_IT_area_BT": "Barletta-Andria-Trani (Puglia)",
+  "sign_up_form_enum_IT_area_FM": "Fermo (Marche)",
+  "sign_up_form_enum_IT_area_MB": "Monza E Brianza (Lombardia)"
```

Comment:
There might be a library handling the different areas in a country (as they're not translated). Could it make sense ? (As it's likely not going to change)

Reply:
It's actually likely to change you mean !

"Nord-Pas-de-Calais" + "Picardie" === "Hauts-de-France"

So yes, we should use a library to manage the areas or an API that can map our codes to areas

Reply:
Indead, it can be a good idea. For now I let like this but I plan to check a lib that handle that. I hope that area code from APIs are ISO :-)